lib/std/tempfiles.nim

```diff
@@ -82,13 +82,21 @@ template randomPathName(length: Natural): string =
     res[i] = state.sample(letters)
   res
 
+proc genTempPath*(prefix, suffix: string, dir = ""): string =
```

Comment:
- Can't `createTempDir` use it too?
- add inline pragams

Reply:
good idea, done